conf/application.conf

```diff
@@ -110,12 +110,30 @@ microservice {
       }
       template_id = "dce5ac8a-0970-41a0-b993-bde1beab5825"
       host = "http://localhost:6099"
+      circuit-breaker {
+        service-name = "GovNotify service"
```

Comment:
these look like the same values as what we have in app-config-base, do we not want them to be different in order to faciliate local testing?

Reply:
Not really they will suffice. We can always change them.

Reply:
OK, but if they are the same as the entries in app-config-base then we can remove those entries from app-config-base

Reply:
Basically one might want to change these locally if need be so hence why I left them here and then they will be different to app-config-base